src/components/EquippedCharacterSheetSlot/index.tsx

```diff
@@ -80,4 +79,9 @@ const AttributesContainer = styled.div`
   gap: 4px;
 `;
 
+const NameBox = styled.span`
+  max-width: 96px;
```

Comment:
Only setting max-width means that a smaller name will have a smaller div right? If so, the content after the name may not be aligned if it's a short name, that will be fine but may lead to some subtle jank in the UI where similar elements aren't lined up.